schema/artwork/index.js

```diff
@@ -32,6 +34,17 @@ const ArtworkType = new GraphQLObjectType({
       _id: {
         type: GraphQLString,
       },
+      to_s: {
+        type: GraphQLString,
+        resolve: ({ artist, title, date, partner }) => {
+          return _.compact([
+            (artist && artist.name),
+            (title && `‘${title}’`),
+            date,
+            (partner && partner.name),
+          ]).join(', ');
+        },
+      },
```

Comment:
But if we want to have display fields we will mostly have more than just a single field that `to_s` could handle. How would you handle those?